specification/resources/registry/examples/python/multiregistry_create.yml

```
@@ -0,0 +1,13 @@
+lang: Python
+source: |-
+  import os
+  from pydo import Client
+
+  client = Client(token=os.environ.get("DIGITALOCEAN_TOKEN"))
+
+  req = {
+    "name": "example",
+    "region": "fra1"
+  }
+
+  resp = client.registry.create(body=req)
```

```suggestion
  resp = client.registries.create(body=req)
```

As mentioned, these are generated from the operation ID. So at the moment, that would actually be `client.multiregistry.create`, but I think we really want `client.registries.create`.

---

Got you @andrewsomething. Just for my information, is the python client generated automatically?

---

:wave: Python client is generated automatically as soon as `godo` changes merged to godo `main` branch.

---

Hey @andrewsomething @loosla 👋, we are back at this work.

> As mentioned, these are generated from the operation ID. So at the moment, that would actually be client.multiregistry.create, but I think we really want client.registries.create.

Just to confirm: if we want the python client to generate like `client.registries.xxx`, it is sufficient to change the `operationId`. I believe I should also change the filename to maintain consistency.

---

@nayanjd-do yes that is correct changing the `operationId` should be sufficient and yes changing the filename would be to a good choice too